pkg/multiboot/multiboot.go

```diff
@@ -76,7 +77,7 @@ type MemoryMap struct {
 	Type uint32
 }
 
```

Comment:
Do we need both MemoryMap and MemoryMaps exported?

Reply:
not MemoryMaps, but MemoryMap, because multiboot.Description uses MemoryMap